test/browser.js

```diff
@@ -18,7 +19,6 @@ idb.deleteDatabase('ipfs')
 idb.deleteDatabase('ipfs/blocks')
```

Comment:
Lets do that, but as this will touch many repos all over the place lets make this its own set of PRs instead of adding it on top of these migrations